sdk/formrecognizer/tests.yml

```diff
@@ -50,6 +51,24 @@ jobs:
         AZURE_FORM_RECOGNIZER_SELECTION_MARK_STORAGE_CONTAINER_SAS_URL: $(python-formrecognizer-test-selection-mark-storage-sas-url)
         AZURE_FORM_RECOGNIZER_TESTING_DATA_CONTAINER_SAS_URL: $(python-formrecognizer-test-storage-testing-data-sas-url)
         AZURE_FORM_RECOGNIZER_AAD_ENDPOINT: $(python-formrecognizer-test-aad-endpoint)
+        # EnvVars for samples to run. Mostly re-used from above values
```

Comment:
I think this is going in a direction that will make maintaining these very difficult. In other languages we are using arm templates for the samples to avoid the need for all these different static secrets. 

I know that @seankane-msft is working to enable arm template support in the python repo and we should try to start using those for samples as well if at all possible.

Reply:
Oh nice, didn't know that we were moving towards the arm template way in Python. @seankane-msft - what's the status on that? Will that be blocking on this PR?

Reply:
Support for arm templates was added in tests.yml files. To add a deployment of arm templates you'll need a `test-resources.json` file and parameter `DeployArmTemplate: true` to your tests.yml file. The former you can probably take from the .NET/JS/Java sdks.

Reply:
Talked to Sean offline and opened an issue to move Form Recognizer to ARM templates in the near future: #15736. Confirmed that this doesn't block on this PR.